hackathon_site/dashboard/frontend/src/components/general/Header/Header.test.js

```diff
@@ -3,18 +3,20 @@ import { render } from "@testing-library/react";
 import Header from "./Header";
 import { withRouter } from "testing";
 
-test("renders header with navbar", () => {
-    const { getByText, getByTestId } = render(withRouter(<Header />));
-    expect(getByText(/Hackathon Name/i)).toBeInTheDocument();
-    expect(getByTestId("headerLogo")).toBeInTheDocument();
-    expect(getByText("Dashboard")).toBeInTheDocument(); // Checking if navbar is in header
-});
+describe("<Header />", () => {
+    test("renders header with navbar", () => {
+        const { getByText, getByTestId } = render(withRouter(<Header />));
+        expect(getByText(/Hackathon Name/i)).toBeInTheDocument();
```

Comment:
We should probably make "Hackathon Name" a constant somewhere. It's going to be a pain to have to replace it in so many places eventually (if you don't want to do this now, that could be a separate piece of work / PR)